cmd/main.go

```diff
@@ -21,7 +22,15 @@ import (
 	"github.com/bookingcom/bpfink/pkg"
 )
 
-var BuildDate = "(development)" //nolint:gochecknoglobals
+// nolint:gochecknoglobals
+var (
+	BuildDate          = "(development)"
+	Once               sync.Once
```

Comment:
this variable is needed only for metric init and can't be used anywhere else, so no need to have it in global scope (moreover with a such common name), move it to MetricsInitialized struct